crates/matrix-sdk-base/src/store/traits.rs

```diff
@@ -808,6 +808,10 @@ pub enum StateStoreDataValue {
 
     /// A list of recently visited room identifiers for the current user
     RecentlyVisitedRooms(Vec<String>),
+
+    /// Persistent data for
+    /// `matrix_sdk_ui::unable_to_decrypt_hook::UtdHookManager`.
+    UtdHookManagerData(Vec<u8>),
```

Comment:
I could wrap the bytes in a newtype, if you like?

Reply:
Thinking about it more: could it be the type we serialize as bytes itself? All our state stores will (de)serialize this value, which is a bit of a waste since we've already done that elsewhere.

Reply:
Hrm, well, not without making `matrix-sdk-base` depend on `growable-bloom-filter`, which feels a bit odd.

Reply:
Doesn't sound too bad to me; what do other Rust maintainers think? cc @poljar @andybalaam @Hywan

Reply:
I think I prefer the stronger type as well, the dependency will be in the tree anyways as far as I understand, albeit in a slightly different place.